### DISCUSSION TITLE
Can I forward NativeControlHost focus/inputs to Avalonia?

### DISCUSSION THREAD

**Discussion by jgcodes2020** [AvaloniaUI/Avalonia#10967] [Q&A]
I have a `NativeControlHost` that displays some OpenGL content from a third-party library that doesn't support Avalonia's rendering method. However, I want Avalonia to handle any user input that occurs to the window.

To my knowledge, `NativeControlHost` is simply a container control for a native child window, and if the window was transparent to input, the underlying Avalonia control would receive the focus/input instead. So my current idea would be to call `EnableWindow` or its equivalent to make the window input-transparent.

**Comment by maxkatz6** [comment]
Theoretically you can pass events to the InputManager.InputManager.
